flathub.json

```
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
+  "automerge-flathubbot-prs": false
```

```suggestion
  "only-arches": ["x86_64"]
```
Also disappointing, but the bar is low at it's proprietary.